### PR TITLE
Added italian translation

### LINKED ISSUE #13: Add Italian translation
After @Magicianred comment on #12 would be great to have an Italian translation of the `README.md` and for the license texts.

The changes that would be needed are:

- [ ] Add a new `it` folder containing a `README.md` file which should be a translation from the English `README.md`.
- [ ] Add the Italian version of the license texts. 
- [ ] Add a badge on the main `README.md` pointing to the Italian translation folder.
- [ ] Add a badge on `it/README.md` pointing to the main `README.md` (see the other translations).

You don't need to translate the licenses texts, they can be downloaded from Creative Commons website. The last time I checked, only the English version of the license texts could be downloaded as a `.txt` file, so we used to have to reformat the HTML files as `.txt` files. After https://github.com/creativecommons/cc-licenses/pull/56 has been merged, we should be able to download the `txt` files from CC, but I don't know if that code has been deployed yet. I'll ask and comment here if I hear some news.

### PR DESCRIPTION
**Pull request by Magicianred**
Added a new it folder containing a README.md file that is a translation of the English README.md.
Added the Italian version of the license texts (convert the html version of the creative commons website).
Added a badge on the main README.md pointing to the Italian translation folder.
Added a badge on it/README.md pointing to the main README.md.

I hope the formatting of the plain text of the licence is pretty good :) If not, I try to correct it

Thanks

Fixes #13

### CODE REVIEW

**Comment by weensy** [comment]
I don't know Italian, but the formatting seems good!

File: it/README.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+> Tu non dovrai usare le licenze Creative Commons per i programmi.
+> Usa le licenze Free Software per questi tipi di repository, come ad esempio GPL, BSD, ecc.
+> Le licenze Creative Commons sono pensate solo per le opere intellettuali.
+
+Il modo più semplice per trovare la giusta licenza CC per il tuo progetto è il
+[sito Creative Commons](https://creativecommons.org/choose/?lang=it).
+
+Qui vi presento due tipi di licenze CC:
+
+* [Creative Commons Attribution 4.0 International](#cc-attribution-40-international)
```
Comment (santisoler) [inline it/README.md]: I think the names of the licenses should be also translated following the translation made by CC.
```suggestion
* [Creative Commons Attribuzione 4.0 Internazionale](#cc-attribution-40-international)
```

File: it/README.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+* [Creative Commons Attribution-ShareAlike 4.0 International](#cc-attribution-sharealike-40-international)
```
Comment (santisoler) [inline it/README.md]:
```suggestion
* [Creative Commons Attribuzione - Condividi allo stesso modo 4.0 Internazionale](#cc-attribution-sharealike-40-international)
```

File: it/README.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+
+Ulteriori informazioni sulle licenze ed i testi delle licenze formattate in ```plain text``` possono essere trovate sul sito https://choosealicense.com/.
+
+Se tu vuoi scaricare una versione vettoriale delle immagini Creative Commons ed altre cose relative, visita il sito https://creativecommons.org/about/downloads/.
+
+
+## CC Attribution 4.0 International
```
Comment (santisoler) [inline it/README.md]:
```suggestion
## CC Attribuzione 4.0 Internazionale
```

File: it/README.md [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+[Creative Commons Attribution 4.0 International License][cc-by].
+
+[![CC BY 4.0][cc-by-image]][cc-by]
+
+[cc-by]: http://creativecommons.org/licenses/by/4.0/
+[cc-by-image]: https://i.creativecommons.org/l/by/4.0/88x31.png
+[cc-by-shield]: https://img.shields.io/badge/License-CC%20BY%204.0-lightgrey.svg
+
+
+## CC Attribution-ShareAlike 4.0 International
```
Comment (santisoler) [inline it/README.md]:
```suggestion
## CC Attribuzione - Condividi allo stesso modo 4.0 Internazionale
```

File: it/README.md [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+
+## CC Attribution 4.0 International
+
+Per aggiungere una licenza CC BY al tuo progetto, ti basta aggiunge il seguente testo al tuo file `README.md`. Dovrai inoltre copiare il corrispondente [file di testo della licenza](LICENSE-CC-BY) e rinominarlo in `LICENSE`.
+
+```markdown
+Shield: [![CC BY 4.0][cc-by-shield]][cc-by]
+
+This work is licensed under a
+[Creative Commons Attribution 4.0 International License][cc-by].
```
Comment (santisoler) [inline it/README.md]: Could you translate these sentence also? Remember to change the name of the license

File: it/README.md [hunk 6]
```diff
@@ -0,0 +1,81 @@
...
+
+[cc-by]: http://creativecommons.org/licenses/by/4.0/
+[cc-by-image]: https://i.creativecommons.org/l/by/4.0/88x31.png
+[cc-by-shield]: https://img.shields.io/badge/License-CC%20BY%204.0-lightgrey.svg
+```
+
+Shield: [![CC BY 4.0][cc-by-shield]][cc-by]
+
+This work is licensed under a
+[Creative Commons Attribution 4.0 International License][cc-by].
```
Comment (santisoler) [inline it/README.md]: Same here

File: it/README.md [hunk 7]
```diff
@@ -0,0 +1,81 @@
...
+
+## CC Attribution-ShareAlike 4.0 International
+
+Per aggiungere una licenza CC BY-SA al tuo progetto, ti basterà aggiungere il seguente testo al tuo file `README.md`. Dovrai inoltre copiare il corrispondente [file di testo della licenza](LICENSE-CC-BY-SA) e rinominarlo in `LICENSE`.
+
+```markdown
+Shield: [![CC BY-SA 4.0][cc-by-sa-shield]][cc-by-sa]
+
+This work is licensed under a
+[Creative Commons Attribution-ShareAlike 4.0 International License][cc-by-sa].
```
Comment (santisoler) [inline it/README.md]: Same here

File: it/README.md [hunk 8]
```diff
@@ -0,0 +1,81 @@
...
+
+[cc-by-sa]: http://creativecommons.org/licenses/by-sa/4.0/
+[cc-by-sa-image]: https://licensebuttons.net/l/by-sa/4.0/88x31.png
+[cc-by-sa-shield]: https://img.shields.io/badge/License-CC%20BY--SA%204.0-lightgrey.svg
+```
+
+Shield: [![CC BY-SA 4.0][cc-by-sa-shield]][cc-by-sa]
+
+This work is licensed under a [Creative Commons Attribution-ShareAlike 4.0
+International License][cc-by-sa].
```
Comment (santisoler) [inline it/README.md]: Same here

File: it/README.md [hunk 9]
```diff
@@ -0,0 +1,81 @@
...
+
+```markdown
+Shield: [![CC BY 4.0][cc-by-shield]][cc-by]
+
+This work is licensed under a
+[Creative Commons Attribution 4.0 International License][cc-by].
+
+[![CC BY 4.0][cc-by-image]][cc-by]
+
+[cc-by]: http://creativecommons.org/licenses/by/4.0/
```
Comment (santisoler) [inline it/README.md]: Would be nice that these links point to the Italian translated websites of CC:
```suggestion
[cc-by]: http://creativecommons.org/licenses/by/4.0/deed.it
```

File: it/README.md [hunk 10]
```diff
@@ -0,0 +1,81 @@
...
+```
+
+Shield: [![CC BY 4.0][cc-by-shield]][cc-by]
+
+This work is licensed under a
+[Creative Commons Attribution 4.0 International License][cc-by].
+
+[![CC BY 4.0][cc-by-image]][cc-by]
+
+[cc-by]: http://creativecommons.org/licenses/by/4.0/
```
Comment (santisoler) [inline it/README.md]:
```suggestion
[cc-by]: http://creativecommons.org/licenses/by/4.0/deed.it
```

File: it/README.md [hunk 11]
```diff
@@ -0,0 +1,81 @@
...
+
+```markdown
+Shield: [![CC BY-SA 4.0][cc-by-sa-shield]][cc-by-sa]
+
+This work is licensed under a
+[Creative Commons Attribution-ShareAlike 4.0 International License][cc-by-sa].
+
+[![CC BY-SA 4.0][cc-by-sa-image]][cc-by-sa]
+
+[cc-by-sa]: http://creativecommons.org/licenses/by-sa/4.0/
```
Comment (santisoler) [inline it/README.md]:
```suggestion
[cc-by-sa]: http://creativecommons.org/licenses/by-sa/4.0/deed.it
```

File: it/README.md [hunk 12]
```diff
@@ -0,0 +1,81 @@
...
+```
+
+Shield: [![CC BY-SA 4.0][cc-by-sa-shield]][cc-by-sa]
+
+This work is licensed under a [Creative Commons Attribution-ShareAlike 4.0
+International License][cc-by-sa].
+
+[![CC BY-SA 4.0][cc-by-sa-image]][cc-by-sa]
+
+[cc-by-sa]: http://creativecommons.org/licenses/by-sa/4.0/
```
Comment (santisoler) [inline it/README.md]:
```suggestion
[cc-by-sa]: http://creativecommons.org/licenses/by-sa/4.0/deed.it
```

**Review by santisoler — CHANGES_REQUESTED** [review]
Thanks @Magicianred for this awesome work!
I left some minor suggestions, they might seem a lot but they're mostly the same suggestion repeated on every appearance of the same text, so don't worry.

One more thing, since #10 has been merged, would you like to add a translation and its entry to the `README.md`? If I'm asking too much, we can merge this PR first, and take care of this extra addition later.

Thanks again for the contribution!

**Comment by Magicianred** [comment]
I made all changes and I added the cc-by-nc-sa in italian :)
Thanks :)

File: it/README.md [hunk 13]
```diff
@@ -0,0 +1,110 @@
...
+> Tu non dovrai usare le licenze Creative Commons per i programmi.
+> Usa le licenze Free Software per questi tipi di repository, come ad esempio GPL, BSD, ecc.
+> Le licenze Creative Commons sono pensate solo per le opere intellettuali.
+
+Il modo più semplice per trovare la giusta licenza CC per il tuo progetto è il
+[sito Creative Commons](https://creativecommons.org/choose/?lang=it).
+
+Qui vi presento tre tipi di licenze CC:
+
+* [Creative Commons Attribuzione 4.0 Internazionale](#cc-attribution-40-international)
```
Comment (santisoler) [inline it/README.md]:
```suggestion
* [Creative Commons Attribuzione 4.0 Internazionale](#cc-attribuzione-40-internazionale)
```

File: it/README.md [hunk 14]
```diff
@@ -0,0 +1,110 @@
...
+
+* [Creative Commons Attribuzione - Condividi allo stesso modo 4.0 Internazionale](#cc-attribution-sharealike-40-international)
```
Comment (santisoler) [inline it/README.md]:
```suggestion
* [Creative Commons Attribuzione - Condividi allo stesso modo 4.0 Internazionale](#cc-attribuzione-condividi-allo-stesso-modo-40-internazionale)
```

File: it/README.md [hunk 15]
```diff
@@ -0,0 +1,110 @@
...
+
+* [Creative Commons Attribuzione - Non commerciale - Condividi allo stesso modo 4.0 Internazionale](#cc-attribution-noncommercial-sharealike-40-international)
```
Comment (santisoler) [inline it/README.md]:
```suggestion
* [Creative Commons Attribuzione - Non commerciale - Condividi allo stesso modo 4.0 Internazionale](#cc-attribuzione-non-commerciale-condividi-allo-stesso-modo-40-internazionale)
```

**Review by santisoler — CHANGES_REQUESTED** [review]
Again, awesome work @Magicianred  !

I just noticed that the link in the `README.md` to the following sections were broken. I left some suggestion to fix them, but they need to checked, high change of being broken too 😬 

Besides that, I think this PR should be ready to be merged.
Thanks again for your time!

**Comment by Magicianred** [comment]
thanks, i didn't notice
I have now fixed the links :)

**Comment by santisoler** [comment]
Awesome @Magicianred ! Thanks again for the contribution.

I'm updating this branch and merging it! 🎉  🚀 🏅

**Review by santisoler — APPROVED** [review]

**Comment by Magicianred** [comment]
Thanks you too :)

**Pull request merged**